### PR TITLE
Fix mapping

### PR DESCRIPTION
**Pull request by jacopocarlini**
this PR depends on #54 

#### List of Changes

- bug fixing entities mapping
- refactoring responses 
- added response validator
- changed password of the Docker DB

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/pagopa/apiconfig/config/ResponseValidator.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+    /**
+     * This method validates the response annotated with the {@link javax.validation.constraints}
+     *
+     * @param joinPoint not used
+     * @param result the response to validate
+     */
+    @AfterReturning(pointcut = "execution(* it.pagopa.pagopa.apiconfig.controller.*.*(..))", returning = "result")
+    public void validateResponse(JoinPoint joinPoint, Object result) {
```
Comment (pasqualespica) [inline src/main/java/it/pagopa/pagopa/apiconfig/config/ResponseValidator.java:32]: why did you add this ?
To verify `@NotNull` field ?

**Review by pasqualespica — APPROVED** [review]
great 👍

File: src/main/java/it/pagopa/pagopa/apiconfig/config/ResponseValidator.java (hunk 1 above)
Comment (jacopocarlini) [reply src/main/java/it/pagopa/pagopa/apiconfig/config/ResponseValidator.java:32]: Yes, to verify `@NotNull` in the responses. In this way, we can detect immediately if our response doesn't follow the swagger definition (ex: description null).

Comment (jacopocarlini) [reply src/main/java/it/pagopa/pagopa/apiconfig/config/ResponseValidator.java:32]: Yes, to verify Annotation in the response class
